pom.xml

```diff
@@ -156,19 +158,7 @@
             <plugins>
                 <plugin>
                     <artifactId>maven-compiler-plugin</artifactId>
-                    <configuration>
-                        <source>1.8</source>
-                        <target>1.8</target>
-                        <compilerVersion>1.8</compilerVersion>
-                        <showDeprecation>false</showDeprecation>
-                        <showWarnings>true</showWarnings>
-                        <optimize>true</optimize>
-                        <fork>true</fork>
-                        <compilerArguments>
-                            <source>1.8</source>
-                            <target>1.8</target>
-                        </compilerArguments>
-                    </configuration>
+                    <version>3.11.0</version>
```

Comment:
Could you check if those plugins are defined in jboss-parent? If so, you can simply overwrite properties instead. Makes it a bit easier to manage all version in a single place.

Reply:
ok

Reply:
jboss-parent uses a very old / deprecated configuration syntax (source/target elements) and others for `maven-compiler-plugin` which generates warnings so it had to be overridden.
I will try to remove some of the duplication vs. jboss-parent though

Reply:
Actually, it wasn't jboss-parent that was old / outdated but Graphene pom itself. It is now fixed by removing plugins from it and thus taking the ones from jboss-parent.